lit_gpt/config.py

```diff
@@ -781,6 +781,44 @@ def norm_class(self) -> Type:
         configs.append(copy)
 
 
+#################
+# Allen AI OLMo
+#################
+olmo = [
+    # https://huggingface.co/allenai/OLMo-1B/blob/main/config.json
+    dict(
+        name="OLMo-1b-hf",
+        hf_config=dict(org="allenai", name="OLMo-1b"),
+        vocab_size=50280,
+        padding_multiple=64,  ## TODO: Not sure how to determine this
```

Comment:
You can use this function to find the `padded_vocab_size`:
https://github.com/Lightning-AI/lit-gpt/blob/01ae9d37ef464b90de2aa6d00df3c985c65cbc56/lit_gpt/utils.py#L25-L29

where `n` is the vocab size and `k` - padding multiple.

If I'm not mistaken, the reason for it originates from the [tweet](https://twitter.com/karpathy/status/1621578354024677377?lang=en) by Andrej Karpathy.

So if you enter `vocab_size` and `padding_multiple` - the code will find the closest efficient size.
Or simply set the size of vocabulary as `padded_vocab_size` and padding multiple will not have any effect.

Reply:
oh that makes total sense now! Thanks you!